Create folder design with all mockups (#54)
KrispyTech/airneis: docs/design/mockups/web_mobile/homepage.md

# Home Page Mockup

![Home Page Mockup](./images/homepage.png)

> The home page is the main page that customers access when they visit the site or on their mobile phone.

Reviewer: ```suggestion
The home page is the main page that customers access when they visit from our website or our mobile phone.
```

> The first part of the mackup is "The Highlanders of the moment":
> - It represents the current categories that have been determined in the back office.
> 
> The second part of the mackup represents the category cards:
> - Each card contains an image and the category title.
> 

Reviewer: ```suggestion
The first part of the mock-up is "The Highlanders of the moment":
- It represents the current categories that have been determined in the back office.

The second part of the mock-up represents the category cards:
- Each card contains the category's image and the title.
```
